resources/css/my-index.css

```diff
@@ -0,0 +1,301 @@
+  margin: 7% 9% 2% 8%;
+  width: 84%;
+}
+
+.card {
+  text-align: center;
+}
+.container {
+  font-weight: bold;
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+    Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+  padding: 6% 5%;
+  font-size: 112%;
+  color: rgb(115 116 145);
+}
+#explore-auroville {
+  border-radius: 50%;
+  width: 64%;
+  height: 14em;
+}
+#partner-with-auroville {
+  border-radius: 50%;
+  width: 59%;
+  height: 14em;
+}
+#support-auroville {
+  border-radius: 50%;
+  width: 69%;
+  height: 14em;
+}
+#overview {
+  text-align: center;
+  font-size: 235%;
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+    Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+  color: rgb(55, 65, 81);
+}
+#overview-content {
+  text-align: center;
+  margin: 1% 20% 7% 20%;
+  background-color: rgb(255, 247, 237);
+  padding: 2% 3%;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+}
+
+.flex {
+  display: flex;
+  flex-direction: row;
+  text-align: center;
+}
+.flex1 {
+  padding-bottom: 9px;
+  gap: 8% 8%;
+}
+.flex2 {
+  gap: 8% 8%;
+}
+#overview-content div {
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+    Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+  font-weight: bold;
+  font-size: 172%;
+  color: rgb(245, 158, 12);
+  width: 100%;
+}
+#overview-content p {
+  font-size: 16px;
+  color: rgb(107, 115, 129);
+  margin-top: -13px;
+}
+#seven {
+  text-align: center;
+  font-size: 230%;
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+    Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+}
+.grid-2 {
+  display: grid;
+  grid-template-columns: auto auto auto;
+  gap: 4% 3%;
+  margin: 1% 11% 10% 11%;
+}
+.grid-2 a {
+  text-decoration: none;
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+    Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+  color: rgb(155 162 173 / 85%);
+  font-weight: bold;
+}
+.grid-2 h4 {
+  color: rgb(22, 30, 53);
+  margin: 3px;
+  font-size: 107%;
+}
+.container-2 p {
+  margin-top: 0px;
+  font-size: 110%;
+  font-weight: 500;
+}
+.grid-2 img {
+  border-radius: 10px;
+}
+#silhouette {
+  margin-bottom: -7px;
+}
+#footer {
+  display: grid;
+  grid-template-columns: auto auto auto auto;
+  color: rgb(211, 192, 192);
+  background-color: rgb(0, 0, 0);
+  padding-left: 3%;
+  padding-bottom: 3%;
+  line-height: 12px;
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+    Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+}
+#footer h4 {
+  font-size: 15px;
+}
+#footer a {
+  color: rgb(160, 153, 153);
+  text-decoration: none;
+  line-height: 12px;
+}
+#footer a:hover {
+  color: rgb(255, 255, 255);
+  cursor: pointer;
+}
+#b {
+  padding-left: 10%;
+}
+#c {
+  padding-left: 10%;
```

Comment:
Write meaningful names. if both #b and #c have same padding, why don't you use class?

Reply:
ok done !